bodhi/server/templates/fragments.html

```diff
@@ -77,7 +77,7 @@
 </%def>
 
 <%def name="update(update, display_user=True, display_request=False, display_release=True, display_karma=True)">
-  <td class="stretch-table-column">${util.update2html(update) | n}</td>
+  <td class="stretch-table-column">${util.update2html(update) | n} ${util.type2icon(update['type'])|n} </td>
```

Comment:
yes! util.type2icon is the same call we make to render the the top icons on the update page, so the same title will be there. Reuse FTW!. Here is a screenshot for good measure:

![screenshot from 2017-04-04 18-39-20](https://cloud.githubusercontent.com/assets/592259/24648241/62c523ba-1966-11e7-9ba9-c6e1c1eb8ec5.png)